screens/home.js

```diff
@@ -0,0 +1,80 @@
+import React, { useEffect, useState } from 'react'
+import { View, ScrollView, Dimensions } from 'react-native'
+import { Text, List, ActivityIndicator } from 'react-native-paper'
+import { RFValue } from 'react-native-responsive-fontsize'
+
+import PageTemplate from '../components/page-template'
+import i18n from '../lib/i18n'
+import Backend from '../lib/backend'
+import TeamItem from '../components/team-item'
+import { getUserGivenName } from '../lib/auth'
+
+export default function HomeScreen({ route, navigation }) {
+  const { authToken } = route.params
+  const [events, setEvents] = useState(null)
+
+  useEffect(() => {
+    Backend.fetchTeams(route.params.authToken).then(teams => {
+      const events = teams.reduce((obj, team) => {
+        if (!obj[team.event.name]) obj[team.event.name] = []
+        obj[team.event.name].push(team)
+        return obj
+      }, {})
+      setEvents(events)
+    })
+  }, [])
+
+  const width = Dimensions.get('window').width
+
+  const handleTeamSelect = item => navigation.navigate('PRE_INST', { item, authToken })
+
+  return (
+    <PageTemplate>
+      {!events ? (
```

Comment:
Notice the difference in the way you write the conditions. Here you write it within the jsx using the `condition ? onTrue : onFalse` syntax, but in `pre-instructor.js`  you write it as an `if () { } else { }`.
This is not something that will prevent the merge, but you should try to keep the language style consistent.

Reply:
In ``PreInstructorScreen``, the root component is different between the 2 states (``<PageTemplate />`` vs ``<PageTemplate hideHeader>``), while in ``HomeScreen``, the root component is the same.